index.bs

```diff
@@ -161,19 +162,16 @@ To <dfn abstract-op local-lt="parse header">obtain a response's embedder policy<
 
 3.  If |header| is not `null`:
 
-    1.  Set |header| to the result of [=isomorphic decoding=] |header|.
+    1.  Let |parsed policy| and |parsed parameters| be the result of executing
+        [$Parsing Header Fields into Structured Headers$] with <var ignore>input_string</var> set to
```

Comment:
This algorithm returns a structured header value, no? Why do you expect two return values?

Reply:
As currently written, https://tools.ietf.org/html/draft-ietf-httpbis-header-structure-14#section-4.2 returns the output of https://tools.ietf.org/html/draft-ietf-httpbis-header-structure-14#section-4.2.3, which returns "a (bare_item,
   parameters) tuple". I agree that this isn't entirely clear from the document.

Reply:
Well, I'd recommend treating it as a tuple in that case and not treating it as multiple return values. (Both of these things exist in other specifications as independent concepts.)

Reply:
Along the lines of "Let |parsed| be the (bare_item, parameters) [=tuple=] resulting from executing ..."?

(It would be nice if we could define these in a way bikeshed could pick up. Perhaps we can add some sort of "Integration with Structured Headers" section to Fetch?)

Reply:
Yeah, as mentioned in my overall review comment I think we should add this to Fetch. In particular, add an algorithm that takes a header list, name, and an expected type? And perhaps the expected type is optional in case you support multiple different ones if that's a thing?

Reply:
Ha. I missed the comment where you said all the things I've said. Sorry about that. I'll stop loudly agreeing with you and try to send you a patch instead.

Reply:
https://github.com/whatwg/fetch/pull/983.